config/config-feature-flags.yaml

```diff
@@ -125,7 +125,7 @@ data:
   enable-cel-in-whenexpression: "false"
   # Setting this flag to "true" will enable the use of StepActions in Steps
   # This feature is in preview mode and not implemented yet. Please check #7259 for updates.
-  enable-step-actions: "false"
+  enable-step-actions: "true"
```

Comment:
disable this when submitting the final PR :) before removing the hold/draft.

Reply:
sure, thank you for the reminding